extensions/github-review-requests/src/hooks/useSearch.ts

```diff
@@ -0,0 +1,97 @@
+import { preferences } from "@raycast/api";
+import { GraphQLClient } from "graphql-request";
+import { useEffect, useState } from "react";
+import { StatusState } from "../schema.generated";
+import { formatDate } from "../util";
+
+import { getSdk } from "./SearchReviewRequest.generated";
+
+const api = getSdk(
+  new GraphQLClient("https://api.github.com/graphql", {
+    headers: {
+      Authorization: `token ${preferences.token.value}`,
+    },
+  })
+);
+
+type CommitStatus = "pending" | "success" | "failure";
+
+export type PullRequest = {
+  title: string;
+  authorAvatarUrl: string | null;
+  repository: string | null;
+  updatedAt: string;
+  status: CommitStatus | null;
+  url: string;
+};
+
+export const useSearch = (searchText: string | undefined) => {
+  const [result, setResult] = useState<PullRequest[]>([]);
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [loading, setLoading] = useState(true);
```
Otherwise [empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states)